tree/dataframe/inc/ROOT/RDF/RInterface.hxx

```diff
@@ -1331,6 +1333,13 @@ public:
                                                  const ColumnNames_t &columnList,
                                                  const RSnapshotOptions &options = RSnapshotOptions())
    {
+      // TODO: Remove before releasing 6.40.00
+      static std::once_flag once;
+      std::call_once(once, []() {
+         R__LOG_INFO(ROOT::Detail::RDF::RDFLogChannel())
+            << "The default compression settings of Snapshot has been changed from 101 (ZLIB with compression level 1) "
+               "to 505 (ZSTD with compression level 5).";
+      });
```

Comment:
Since the flag is a function static anyway, this will be slightly more performance (half the number of synchronization before (potoential/maybe) optimization):

```suggestion
      [[maybe_unused]] static bool once = []() {  
         R__LOG_INFO(ROOT::Detail::RDF::RDFLogChannel())
            << "The default compression settings of Snapshot has been changed from 101 (ZLIB with compression level 1) "
               "to 505 (ZSTD with compression level 5).";
         return true;
      } ();
```